react/components/NewChannelForm/index.js

```diff
@@ -52,8 +64,11 @@ class NewChannelForm extends Component {
     this.setState({ visibility });
 
   handleAuthor = (group_id) => {
-    if (group_id === 0) return;
-    this.setState({ group_id });
+    if (group_id === 0) {
```

Comment:
I don't really recall this `group_id === 0` thing — it's a work around?

Reply:
Yeah, was there before. Looks like the mutation accepts a group_id rather than a `MemberInputType`.

Reply:
Ah I see — Weird I have no idea why I made the mutation like that.